mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -1896,6 +1911,7 @@ int CMomentumGameMovement::TryPlayerMove(Vector *pFirstDest, trace_t *pFirstTrac
                 float offsets[] = {(bumpcount * 2) * -sv_ramp_initial_retrace_length.GetFloat(), 0.0f,
                                    (bumpcount * 2) * sv_ramp_initial_retrace_length.GetFloat()};
                 int valid_planes = 0;
+                valid_plane.Init(0.0f, 0.0f, 0.0f);
```

Comment:
If I recall correctly, the default params of Init() is already 0.0f.

Reply:
I like being explicit since we are doing += on it afterwards and not just =.